docs/docs/basic-usage.md

```diff
@@ -64,6 +64,24 @@ $ poetry add pendulum
 It will automatically find a suitable version constraint **and install** the package and subdependencies.
 
 
+## Using your virtual environment
+
+By default, poetry creates a virtual environment in `{cache-dir}/virtualenvs` (`{cache-dir}\virtualenvs` on Windows).
+There are several ways to run commands within this virtual environment.
+
+### Using `poetry run`
+
+To run your script simply use `poetry run python your_script.py`.
+Likewise if you have command line tools such as `pytest` or `black` you can run them using `poetry run pytest`.
+
+### Activating the virtual environment
+
+The easiest way to activate the virtual environment is `poetry shell`.
+
+Alternatively you can manually activate the virtual environment by running `source {path_to_venv}/bin/activate` (`source {path_to_venv}\Scripts\activate.bat` on Windows).
```

Comment:
Might be good to move this to a table; and also perhaps referencing `source $(poetry env info --path)/bin/activate` for convinience. And maybe mention that this is required if you do not want a new sub-shell.

Reply:
I added that one-liner and the table.